modules/cloudsql-instance/variables.tf

```diff
@@ -139,3 +139,9 @@ variable "users" {
   type        = map(string)
   default     = null
 }
+
+variable "ipv4_enabled" {
+  description = "Allow public access to database instance."
```

Comment:
This adds a public IP to the instance. Can you update the description to reflect that?

Reply:
Hi @juliocc, yes i do.
Thanks for the help